text/0054-steering-committee-elections.md

```diff
@@ -0,0 +1,158 @@
+   who is qualified as a Steering Committee voter as outlined in that section.
+
+##### Voter Eligibility
+
+1. Any individual who has contributed to Paketo Buildpacks in the twelve months
+   prior to the election is eligible to vote in the Steering Committee
+   election.
+2. Contributions include, but are not limited to, opening PRs, reviewing and
+   commenting on PRs, opening and commenting on issues, writing design docs,
+   commenting on design docs, participating in mailing list discussions and
+   participating in working groups.
+3. Each election cycle, an initial set of voters will be identified through
+   automated reporting. Any individual who has at least 10 measurable
+   contributions in the last 12 months will be automatically added to the
+   eligible voter list.
+4. If a community member has contributed over the past year but is not captured
+   in automated reporting, they will be able to submit an eligibility form to
+   the current Steering Committee who will then determine whether this member
+   will be eligible. In a case where the Steering Committee declines an
+   eligibility request, the requestor may appeal that decision to the Cloud
+   Foundry Governing Board.
+
+### Implementation
+
+The election will follow a modified version of the existing Cloud Foundry TOC
+Election process and implementation outlined in the [Cloud Foundry Foundation
+Charter](https://github.com/cloudfoundry/community/blob/main/governing-board/charter.md)
+Section 7.e.iii. The modified process is outlined below.
+
+#### Election Method and Tools
+
+1. If the number of candidates is equal to or less than the number of Steering
+   Committee seats available to be elected, the candidates shall be approved
+   after the nomination period has closed.
+2. If there are more Qualified Nominees than open Steering Committee seats
+   available for election, all eligible voters shall elect the Steering
+   Committee members using a time-limited
+   [Condorcet](https://civs.cs.cornell.edu/rp.html) ranking using the Schulze
+   method.
+3. Elections may be run either via the [Condorcet Internet Voting
+   Service](https://civs1.civs.us/) or via a deployment of
+   [Elekto](https://elekto.dev/) administered by the CFF staff.
+
+#### Steering Committee Member Resignations or Removal
+
+The process for handling Steering Committee member resignations or removal will
+align to the rules, with some modifications, outlined in the [Cloud Foundry
+Foundation
+Charter](https://github.com/cloudfoundry/community/blob/main/governing-board/charter.md)
+Sections 7.f. Their modified contents are outlined below.
+
+1. In the case where a Steering Committee member resigns or is removed from the
+   Steering Committee by the Governing Board, the remaining members of the
+   Steering Committee will determine whether the vacant seat either will remain
+   vacant until the next annual election cycle, or will be filled by using the
+   results of the latest election.
```

Comment:
Is this sufficient for a group of three? The CF TOC has five members, so there is more redundancy.

With a group of three, you could run into issues like:

1. Two resignations around the same time, leaving only one member left.
2. A single resignation, but no agreement on how to proceed (since there's an even number of members left).

I'm also not a fan of how it's arbitrarily left up to the remaining steering committee members to decide how to proceed. 

Perhaps we need something more concrete like "if it's within X months of the previous election, the remaining Steering committee members may fill the vacancy based on the previous election results"?

Perhaps we also need something like "If a resignation results in reducing the Steering Committee to only one member, then the remaining member will work with the CFF Governing Board (or TOC?) to determine a course of action."?

What do folks think?

Reply:
I've modified this to state the the Steering Committe, in whatever capacity it remains, will work with the Governing Board to determine next steps.